mesonbuild/interpreterbase/decorators.py

```diff
@@ -394,7 +393,6 @@ def __init__(self, name: str,
                  deprecated: T.Optional[str] = None,
                  deprecated_message: T.Optional[str] = None,
                  deprecated_values: T.Optional[T.Dict[T.Union[_T, T.Type[T.List], T.Type[T.Dict]], T.Union[str, T.Tuple[str, str]]]] = None,
```

Comment:
```suggestion
               deprecated_values: T.Union[T.Dict[T.Union[_T, type, ContainerTypeInfo], T.Union[str, T.Tuple[str, str]]], None, _NULL_T] = _NULL,
```

And `sinve_values` as well.

Reply:
done